<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vignere</class>
 <widget class="QMainWindow" name="Vignere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textMessage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(149, 149, 149);
border-color: rgb(4, 4, 4);
font: 8pt &quot;Franklin Gothic Book&quot;;</string>
    </property>
    <property name="text">
     <string>PlainText/CipherText</string>
    </property>
   </widget>
   <widget class="QLabel" name="key">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(149, 149, 149);
border-color: rgb(4, 4, 4);
font: 8pt &quot;Franklin Gothic Book&quot;;</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Encrypt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textOut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textKey">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_out">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(149, 149, 149);
border-color: rgb(4, 4, 4);
font: 8pt &quot;Franklin Gothic Book&quot;;
</string>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushExit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>180</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 0, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>320</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 214, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0) ;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVignere">
    <property name="title">
     <string>Vigenere </string>
    </property>
   </widget>
   <addaction name="menuVignere"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
